client.go

```diff
@@ -70,26 +78,40 @@ type Client struct {
 }
 
 // NewClient returns a new API Client using the given APIKey
-func NewClient(apiKey string) *Client {
+func NewClient(apiKey string, options ...ClientOptions) *Client {
```

Comment:
The `options` here does not make sense as an array. I understand that the splat allows it to be not specified and thus not change the signature, but I think this would be better with a new function:

```suggestion
func NewClient(apiKey string) *Client {
    return NewClientWithOptions(apiKey, nil)
}

// NewClientWithOptions returns a new API Client using the given APIKey and options
func NewClientWithOptions(apiKey string, options *ClientOptions) *Client {
```

Or, make this a breaking change since a nullable options makes more sense than an array of options. That signature would look like:

```golang
func NewClient(apiKey string, options *ClientOptions) *Client {
```

Reply:
making a nullable options will break clients that is already using the library. I think that your suggest to use a new function will fit better.

I also think to use US as default for newClient instead of nil in that case client don't need to change their code.